src/primitives/basemeasure.jl

```diff
@@ -0,0 +1,46 @@
+
+export basemeasure
+
+import MeasureTheory
+
+function MeasureTheory.basemeasure(c::ConditionalModel{A,B,M}, x=NamedTuple()) where {A,B,M}
+    _basemeasure(M, Model(c), argvals(c), observations(c), x)
```

Comment:
Discuss if this is the right signature? One could make it a function of a Model too, right?

Reply:
Let's assume not, it will often depend on the arguments